utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -40,4 +40,8 @@ def host_target():
         if machine == 'amd64':
             return 'freebsd-x86_64'
 
+    elif system == 'CYGWIN_NT-10.0':
```

Comment:
Just curious, what is the `10.0` version?  Does it change with, say, Windows releases?